### PR TITLE
feature: run prompt in batch

### PR DESCRIPTION
**Pull request by geclos**

### CODE REVIEW

File: apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/index.tsx [hunk 1]
```diff
@@ -59,23 +108,8 @@ export function EvaluationResults({
...
-          { ...args.row, createdAt, realtimeAdded: true },
-          ...(prevData ?? []),
-        ]
-      },
-      { revalidate: false },
-    )
-  }
 
-  useSockets({ event: 'evaluationResultCreated', onMessage })
+  useEvaluationResultsSocket(evaluation, document, mutate)
```
Comment (andresgutgon) [inline apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/index.tsx:112]: is this fixing a bug?

File: apps/web/src/middlewares/errorHandler.ts [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 import { NextRequest, NextResponse } from 'next/server'
 
 export function errorHandler(handler: any) {
   return async (req: NextRequest, res: NextResponse) => {
     try {
       return await handler(req, res)
     } catch (error) {
+      if (env.NODE_ENV === 'development') {
+        console.error(error)
+      }
```
Comment (andresgutgon) [inline apps/web/src/middlewares/errorHandler.ts:12]: use `debug`. This way don't need to check env

File: apps/web/src/stores/documentLogs.ts [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    page: number | null
+    pageSize: number | null
+  },
+  { fallbackData }: SWRConfiguration = {},
+) {
+  const { toast } = useToast()
+  const fetcher = useCallback(async () => {
+    try {
+      const response = await fetch(
+        `/api/projects/${projectId}/commits/${commitUuid}/documents/${documentUuid}/documentLogs?page=${page ?? undefined}&pageSize=${pageSize ?? undefined}`,
```
Comment (andresgutgon) [inline apps/web/src/stores/documentLogs.ts:28]: Add a Fixme. Carlos is doing API routes central constant

**Review by andresgutgon — APPROVED** [review]

File: apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/index.tsx (hunk 1 above)
Comment (geclos) [reply apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/index.tsx:112]: nah just moving code around

File: apps/web/src/middlewares/errorHandler.ts (hunk 2 above)
Comment (geclos) [reply apps/web/src/middlewares/errorHandler.ts:12]: pues tiene razon

**Pull request merged**